orga/tests/integration/components/routes/authenticated/sco-students/list-items-test.js

```diff
@@ -314,6 +314,11 @@ module('Integration | Component | routes/authenticated/sco-students | list-items
         test('it should display import CSV file button', async function(assert) {
           assert.contains('Importer (.csv)');
         });
+
+        test('it should display download template csv button', async function(assert) {
```

Comment:
il manque un test `when organization is SCO, it should not display download template csv button`